src/CalculatorUnitTests/RationalTest.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include <CppUnitTest.h>
+#include "Header Files/Rational.h"
+#include "Header Files/RationalMath.h"
+
+using namespace CalcEngine;
+using namespace CalcEngine::RationalMath;
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace CalculatorEngineTests
+{
+    TEST_CLASS(RationalTest)
+    {
+    public:
+        TEST_CLASS_INITIALIZE(CommonSetup)
+        {
+            ChangeConstants(10, 128);
+        }
+        TEST_METHOD(ModuloTest)
+        {
+            Rational rat25(25);
+            Rational ratminus25(-25);
+            Rational rat4(4);
+            Rational ratminus4(-4);
+            Rational res = Mod(rat25, rat4);
+            VERIFY_ARE_EQUAL(res.ToString(10, FMT_FLOAT, 128), L"1");
+            res = Mod(rat25, ratminus4);
```

Comment:
`Rational` types can be implicitly constructed from `int` values, meaning we can write simpler and more readable code such as:
```suggestion
            res = Mod(25, -4);
```
Assuming you update these test to take advantage of the implicit conversion, is there a need for the named `Rational` objects you declared above? (`rat25`, `ratminus25`, etc.).

Reply:
I missed the implicit conversion, I will update the tests

Reply:
It was part of the first version of the unit tests, before adding more. But I think we should keep them and use them to check that Mod and Rem don't change the value of operands.